gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/ApplicationEntity.java

```diff
@@ -21,12 +21,18 @@
 import java.util.Date;
 import java.util.Objects;
 import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
 
 /**
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author Nicolas GERAUD (nicolas.geraud at graviteesource.com)
  * @author GraviteeSource Team
  */
+@Getter
+@Setter
+@ToString
```

Comment:
do we want to keep our equals and hashcode ?
if not, we can maybe use `@Data` ?

Reply:
Lombok-generated equals would compare all properties by default.

We could make it compare only ID, using this annotation on class : `@EqualsAndHashCode(onlyExplicitlyIncluded = true)`
And this one on`id`property : @EqualsAndHashCode.Include

WDYT ? Would it more more explicit/comprehensible with or without lombok for this case ?

Reply:
As discussed together, I added that